metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_core.py

```diff
@@ -561,7 +565,7 @@ def load_nodes(self) -> Tuple[List[DBTNode], Dict[str, Optional[str]]]:
         if (
             catalog_version
             and catalog_version.startswith("1.7.")
-            and catalog_version < "1.7.3"
+            and version.parse(catalog_version) < version.parse("1.7.3")
```

Comment:
add some exception handling around this, just in case

Reply:
done